### DISCUSSION TITLE
No yarn.lock in /website

### DISCUSSION THREAD

**Discussion by sozercan** [facebook/docusaurus#6035] [Q&A] [answered]
Is it intentional for not having yarn.lock file committed in https://github.com/facebook/docusaurus/tree/main/website? If so, what's the reason?

**Comment by armano2** [comment, marked as answer]
this repo is a monorepo that uses yarn workspaces
https://github.com/facebook/docusaurus/blob/33de0cdf99d29b7909f55df06aa6309df3f4fa49/package.json#L4-L10
workspace require / uses only one yarn.lock

https://classic.yarnpkg.com/lang/en/docs/workspaces/
